styles.css

```diff
@@ -3,220 +3,284 @@
 @import url("https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&display=swap");
 @import url("https://fonts.googleapis.com/css2?family=Meera+Inimai&display=swap");
 
-@media screen and (min-width: 375px) {
-  .header {
+@media screen and (max-width: 1440px) {
```

Comment:
- [ ] Since the first code reviewer suggested please use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: 768px or wider). it should look like this 👉🏽 `@media screen and (min-width: 768px`

![image](https://user-images.githubusercontent.com/58642949/206819319-ae5bcab5-409c-47f2-9953-a0a065e449cf.png)

![image](https://user-images.githubusercontent.com/58642949/206819340-aedd3ec7-cb74-4b3d-a550-9a9eabc4e6ef.png)